src/main/java/org/jspecify/nullness/Implies.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2022 The JSpecify Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jspecify.nullness;
```

Comment:
This should probably be `org.jspecify` or `org.jspecify.meta` or something like that. I remember a bit of discussion on https://github.com/jspecify/jspecify/issues/160, but I'm not sure how much there has been. I don't know if you want to just pick something for now or if you want to block on putting out a wider call to the group.

Reply:
Good catch!